src/components/LeaderListItem/index.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Avatar } from 'antd';
 import { Link } from 'react-router-dom';
 
 import './style.less';
 import { getRank } from '../../utils';
 
-const LeaderListItem = (props) => {
+interface LeaderListItemProps {
+  exp: number
+  image: string
+  name: string
+  placement: number
+  uuid: string
+  rank: number
+}
```

Comment:
To make it easier to reuse these interfaces, should we consider creating a separate "types" folder for all these? It might make backend's job of typing the API payloads easier, possibly.